lute/cli/commands/lint/init.luau

```diff
@@ -37,13 +37,36 @@ local function loadLintRule(path: path.pathlike): types.LintRule
 		`{path} must return a table with a 'lint' function property`
 	)
 
-	return loaded :: types.LintRule -- We have to cast because we can't assert that loaded.lint is a more specific functio type
+	return loaded :: types.LintRule -- We have to cast because we can't assert that loaded.lint is a more specific function type
```

Comment:
I know this is a comment from before but can you expand on this? It's because we can't assert that `loaded.lint` is specifically, say, a `(foo) -> bar`, only that it's a `function`